bumble/gatt_server.py

```diff
@@ -274,6 +280,7 @@ def add_service(self, service: Service):
 
         # Update the service group end
         service.end_group_handle = self.attributes[-1].handle
+        service.registered = True
```

Comment:
see earlier comment about the mutability of the Service object.

Reply:
Done